Jenkinsfile.download_logs

```diff
@@ -55,7 +55,10 @@ pipeline {
         stage('Create tickets') {
             steps {
                 dir ('assisted-installer-deployment') {
-                    sh "skipper run ./tools/triage/create_triage_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} -v"
+                    withCredentials([file(credentialsId: 'prod-pull-secret', variable: 'PULL_SECRET_FILE')]) {
+                      sh 'cp "${PULL_SECRET_FILE}" .'
```

Comment:
ok, I actually didn't think about the mounting part of this, which makes it a bit ugly IMHO (to copy it to the code dir)
it can also have some undesired affects if we're relying on ``git status/diff`` somewhere

in this case, I think ``PULL_SECRET`` (without specifying a file-path, just the value) might be the best solution here. this way it's easy to pass to the container and yet doesn't have the potential to leak in jenkins logs

Reply:
I'll see how that works out then

Reply:
Ok, now we pass the contents of the pull-secret to skipper via an env variable :+1: